packages/cli/src/utils/verifyManifest.ts

```diff
@@ -0,0 +1,104 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { array, number, object, string, z } from 'zod';
+
+const forbiddenExtensions = ['exe', 'dmg', 'cmd', 'sh', 'bat'];
+const getForbiddenExtensionsErrorMessage = (surfaceName: string) =>
+    `Invalid file extension, \`${surfaceName}.filenameExtension\` can not include "${forbiddenExtensions}".`;
+
+const AssetCreationShape = object({}).optional();
+
+const completeAssetType = z.enum(['audio', 'document', 'image', 'video', 'file', 'embeddedContent']);
+const imageAssetType = completeAssetType.exclude(['audio', 'document', 'video', 'file', 'embeddedContent']);
+
+const iconLibraryFilenameExtension = z.enum(['svg']);
+const logoLibraryFilenameExtension = z.enum(['svg', 'jpg', 'jpeg', 'ai', 'eps', 'png', 'tif', 'tiff']);
+
+export const platformAppManifestSchemaV1 = object({
+    appId: string().refine((value) => value.trim() !== '', {
+        message: '`appId` is required',
+    }),
+    appType: string().refine((value) => ['content-block', 'platform-app', 'theme'].includes(value), {
+        message: '`appType` is required',
+    }),
+    surfaces: object({
+        mediaLibrary: object({
+            assetAction: object({
+                type: array(
+                    completeAssetType.refine((value) =>
+                        ['audio', 'document', 'image', 'video', 'file', 'embeddedContent'].includes(value),
```

Comment:
Why do you need the refine?

Reply:
ahhhh thats how it works xD i simply didnt get it, haha